doc/version_history.rst

```diff
@@ -6,6 +6,10 @@
 Version History
 ###############
 
+v0.9.0
+------
+* Fix fits file header to match Vera C. Rubin format
```

Comment:
The name of our project is `Vera C. Rubin Observatory` so please amend this description.